plugins/inputs/ipmi_sensor/README.md

```diff
@@ -68,24 +89,36 @@ KERNEL=="ipmi*", MODE="660", GROUP="telegraf"
-ipmi_sensor,server=10.20.2.203,unit=rpm,name=fan_1b_tach status=1i,value=1775 1458488465013279896
+ipmi_sensor,server=10.20.2.203,name=uid_light,host=foo.bar.com value=0,status=1i 1517125513000000000
+ipmi_sensor,server=10.20.2.203,name=sys._health_led,host=foo.bar.com status=1i,value=0 1517125513000000000
+ipmi_sensor,server=10.20.2.203,unit=watts,host=foo.bar.com,name=power_supply_1 status=1i,value=110 1517125513000000000
+ipmi_sensor,server=10.20.2.203,host=foo.bar.com,name=power_supply_2,unit=watts status=1i,value=120 1517125513000000000
+ipmi_sensor,server=10.20.2.203,name=power_supplies,host=foo.bar.com value=0,status=1i 1517125513000000000
+ipmi_sensor,server=10.20.2.203,name=fan_1,unit=percent,host=foo.bar.com status=1i,value=43.12 1517125513000000000
 ```
 
+
+When retrieving stats from the local machine (no server specified):
+```
+ipmi_sensor,name=uid_light,host=foo.bar.com value=0,status=1i 1517125513000000000
+ipmi_sensor,name=sys._health_led,host=foo.bar.com status=1i,value=0 1517125513000000000
+ipmi_sensor,unit=watts,host=foo.bar.com,name=power_supply_1 status=1i,value=110 1517125513000000000
+ipmi_sensor,host=foo.bar.com,name=power_supply_2,unit=watts status=1i,value=120 1517125513000000000
+ipmi_sensor,name=power_supplies,host=foo.bar.com value=0,status=1i 1517125513000000000
+ipmi_sensor,name=fan_1,unit=percent,host=foo.bar.com status=1i,value=43.12 1517125513000000000
+```
+
+#### Version 2 Schema
+
 When retrieving stats from the local machine (no server specified):
 ```
-ipmi_sensor,unit=degrees_c,name=ambient_temp status=1i,value=20 1458488465012559455
-ipmi_sensor,unit=feet,name=altitude status=1i,value=80 1458488465012688613
-ipmi_sensor,unit=watts,name=avg_power status=1i,value=220 1458488465012776511
-ipmi_sensor,unit=volts,name=planar_3.3v status=1i,value=3.28 1458488465012861875
-ipmi_sensor,unit=volts,name=planar_vbat status=1i,value=3.04 1458488465013072508
-ipmi_sensor,unit=rpm,name=fan_1a_tach status=1i,value=2610 1458488465013137932
-ipmi_sensor,unit=rpm,name=fan_1b_tach status=1i,value=1775 1458488465013279896
+ipmi_sensor,name=uid_light,entity_id=23.1,status_code=ok,status_desc=ok,host=foo.bar.com value=0 1517125474000000000
+ipmi_sensor,host=foo.bar.com,name=sys._health_led,entity_id=23.2,status_code=ok,status_desc=ok value=0 1517125474000000000
+ipmi_sensor,status_code=ok,unit=watts,status_desc=presence_detected,host=foo.bar.com,name=power_supply_1,entity_id=10.1 value=110 1517125474000000000
+ipmi_sensor,host=foo.bar.com,name=power_supply_2,entity_id=10.2,status_code=ok,unit=watts,status_desc=presence_detected value=125 1517125474000000000
+ipmi_sensor,name=power_supplies,entity_id=10.3,status_code=ok,status_desc=fully_redundant,host=foo.bar.com value=0 1517125474000000000
+ipmi_sensor,unit=percent,status_desc=transition_to_running,host=foo.bar.com,name=fan_1,entity_id=7.1,status_code=ok value=43.12 1517125474000000000
```

Comment:
If you regen this output with Telegraf 1.7, it should sort the tags.  You can leave out the host tag since it is added everywhere.